website/docs/cloud-docs/no-code-provisioning/module-design.mdx

```diff
@@ -46,6 +47,16 @@ For example, you could build modules that satisfy the following well-scoped use
 - Deploying all resources needed for a three-tier web application
 - Deploying a database with constraints on resource allocation and deployment region
 
+### Adjust for New Module Versions
+No-code provisioning desgination is now pinned to a selected version of a module. You must select the version at the time of enabling the module. If no selection is made, it will default to the latest version at the time of enabling. This ensures that you can define variable input options for the intended version and that no-code users provision the most appropriate version. When a newer version of the module becomes avialable, you may want to update the pinned no-code version to the latest.
+
 ### Provide Variable Defaults When Possible
 
-The no-code provisioning workflow prompts users to set values for the module's input variables before creating the new workspace and deploying resources. We recommend setting reasonable defaults when possible to reduce the effort and expertise needed to deploy the module. Remember that the workspace can also access variable values set through [global or project-scoped variable sets](/terraform/cloud-docs/workspaces/variables/managing-variables#variable-sets) in your organization.
+The no-code provisioning workflow prompts users to set values for the module version's input variables before creating the new workspace and deploying resources. We recommend setting reasonable defaults when possible to reduce the effort and expertise needed to deploy the module. Remember that the workspace can also access variable values set through global variable sets in your organization.
+
+### Define Dropdown Options for Variables without Defaults
+
+For variables without defaults, you may still want to limit what the no-code user can input. Define these input options at the time of enabling the module for no-code provisioning. You can do this either from the Terraform Cloud UI, via the [No-Code Provisioing API](), or via the [TFE provider](). The latter method is recommended for those that want to track these variable input options through a code approval process, keeping the configuration as code.
+
+Subsequent changes to the variable input options will be surfaced to new provisions of the module version. If you update the selected module version enabled for no-code provisioning, you may also want to revisit the variables and adjust the defined input values accordingly. 
```

Comment:
I tweaked this to be in active voice. I think it is still correct😅, but please let me know if not:
```suggestion
Any changes to a variable's input options are surfaced when you provision a new module version. If you update the selected module version enabled for no-code provisioning, consider revisiting the variables and adjusting the defined input values accordingly. 
```

Reply:
As this doc is intended for the Admin or module author, I'd like to use part of your suggestion:

Any subsequent changes to a variable's input options are surfaced when no-code users provision a new module version. If you update the selected module version enabled for no-code provisioning, consider revisiting the variables and adjusting the defined input values accordingly.

Reply:
Here is the suggestion you'd like added! This looks good to me too 👍 
```suggestion
Any subsequent changes to a variable’s input options are surfaced when no-code users provision a new module version. If you update the selected module version enabled for no-code provisioning, consider revisiting the variables and adjusting the defined input values accordingly.
```